pkg/proxy/proxy.go

```diff
@@ -349,7 +348,26 @@ func BootProxy(logger *zap.Logger, opt Option, appCmd, appContainer string, pid
 	if opt.Port == 0 {
 		opt.Port = 16789
 	}
+	maxAttempts := 1000
+	attemptsDone:=0
+
 
+    if !isPortAvailable(opt.Port) {
+		logger.Info("Port is already in use, switching to next available port",zap.Uint32("port",opt.Port));
```

Comment:
`Port is already in use, switching to next available port {port : 16789}` this is how the logger looks when you run keploy. It gives a sense that you you found that port is already in use and you switching to port 16789 which is wrong. You can delete this log or correct it . Deleting this log makes more sense for me.